odoo/addons/base/wizard/base_language_install.py

```diff
@@ -9,45 +9,56 @@ class BaseLanguageInstall(models.TransientModel):
     _description = "Install Language"
 
     @api.model
-    def _default_language(self):
+    def _default_lang_ids(self):
         """ Display the selected language when using the 'Update Terms' action
             from the language list view
         """
         if self._context.get('active_model') == 'res.lang':
-            lang = self.env['res.lang'].browse(self._context.get('active_id'))
-            return lang.code
+            return self.env['res.lang'].browse(self._context.get('active_ids') or [self._context.get('active_id')])
         return False
-
-    @api.model
-    def _get_languages(self):
-        return [[code, name] for code, _, name, *_ in self.env['res.lang'].get_available()]
-
-    lang = fields.Selection(_get_languages, string='Language', required=True,
-                            default=_default_language)
+    # add a context on the field itself, to be sure even inactive langs are displayed
+    lang_ids = fields.Many2many('res.lang', 'res_lang_install_rel', 'language_wizard_id', 'lang_id', 'Languages', default=_default_lang_ids, context={'active_test': False})
     overwrite = fields.Boolean('Overwrite Existing Terms',
                                default=True,
                                help="If you check this box, your customized translations will be overwritten and replaced by the official ones.")
-    state = fields.Selection([('init', 'init'), ('done', 'done')],
-                             string='Status', readonly=True, default='init')
+    first_lang_id = fields.Many2one('res.lang', compute='_compute_first_lang_id', help="Used when the user only selects one language and is given the option to switch to it")
+
+    def _compute_first_lang_id(self):
+        for record in self:
```

Comment:
We tend to pre-fill void values in batch then update only relevant records (using filtered(field_name)). In the end it gives the same behavior, but "standard" way of writing this becomes more like
```
def _compute_first_lang_id(self):
    self.first_lang_id = False
    for lang_installer in self.filtered('lang_ids'):
        lang_installer.first_lang_id = first_lang_id.lang_ids[0]
```